### ISSUE TITLE
Bug survivor invisible

### ISSUE THREAD

**Issue by oethanx** [Tsuey/L4D2-Community-Update#16]
Often you will see a survivor player's model not moving in the saferoom but he is actually playing normally, this can happen only to yourself and other players don't see it so yes he is invisible and the model is stuck in the saferoom.

Not sure what causes that but it seems that the only way to fix the problem is to open the console and type "record demo name" then enter "stop" and magically you will see the survivor at his real position, I have no idea why these commands are related to this bug this is quite weird.

**Comment by jacob404** [comment]
This issue is caused by a desync between the server and client and can commonly happen after alt-tabbing out of the game. As far as I know this bug exists in all source games (at least released prior to l4d2) and would require Valve's help to fix.

**Comment by oethanx** [comment]
Would it be possible to refresh frames for clients after X seconds? Executing "record demoname;stop"?

**Comment by shqke** [comment]
Your client just requests a full update during recording.

If theres an issue, it is likely to be looked in mismatching or missing partial updates.

**Comment by A1mDev** [comment]
I don’t think this is a mistake and it cannot be solved. Check your internet, you may be losing a lot of packages, or you often minimize the game to your desktop. The fact is that when you join to the server, the server sends full information about the game, the same thing happens when recording a demo, and then only sends information partially. This made for performance, it's stupid to send complete information to every client. So this is not a bug, and will never be fixed =)

**Comment by oethanx** [comment]
Most players have experienced it. Minimizing game does cause this bug. I don't believe it's fair or sensible to omit it as "not a bug" since clients are running default versions of the game (little to no addons) and still experiencing it. Here are other threads made for the same topic: https://github.com/SirPlease/L4D2-Competitive-Rework/issues/319
